apollo-backend/src/main/java/io/logz/apollo/blockers/SingleRegionBlockerResponse.java

```diff
@@ -0,0 +1,38 @@
+package io.logz.apollo.blockers;
+
+import java.util.List;
+
+public class SingleRegionBlockerResponse {
```

Comment:
You are using this class as a result of `RequestBlockerFunction.shouldBlock(...)` 
I think you might want to use something more general (like RequestBlockerResponse) to not confuse people who will support it in future

Reply:
True. change the name.